MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Tabs.xaml

```diff
@@ -0,0 +1,414 @@
+                            <Setter TargetName="SelectionHighlightBsorder" Property="BorderThickness" Value="0,0,2,0" />
+                        </DataTrigger>
+                        <DataTrigger Binding="{Binding TabStripPlacement, RelativeSource={RelativeSource AncestorType={x:Type TabControl}}}" Value="Right">
+                            <Setter TargetName="SelectionHighlightBsorder" Property="BorderThickness" Value="2,0,0,0" />
+                        </DataTrigger>
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="PrimaryLight">
+                            <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueLightForegroundBrush}" />
+                            <Setter Property="Background" Value="{DynamicResource PrimaryHueLightForegroundBrush}" />
+                        </Trigger>
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="PrimaryMid">
+                            <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueMidForegroundBrush}" />
+                            <Setter Property="Background" Value="{DynamicResource PrimaryHueMidForegroundBrush}" />
+                        </Trigger>
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="PrimaryDark">
+                            <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueDarkForegroundBrush}" />
+                            <Setter Property="Background" Value="{DynamicResource PrimaryHueDarkForegroundBrush}" />
+                        </Trigger>
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="Accent">
+                            <Setter Property="BorderBrush" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
+                            <Setter Property="Background" Value="{DynamicResource SecondaryHueMidForegroundBrush}" />
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="{x:Type TabItem}" x:Key="MaterialDesignTabItemCompact" BasedOn="{StaticResource MaterialDesignTabItem}">
+        <Setter Property="Height" Value="48" />
+    </Style>
+
+    <!--NAVIGATION RAIL-->
+
+    <Style  TargetType="{x:Type TabControl}" x:Key="MaterialDesignNavigatilRailTabControl">
+        <Setter Property="Background" Value="{x:Null}"/>
+        <Setter Property="Foreground" Value="{Binding RelativeSource={RelativeSource AncestorType={x:Type FrameworkElement}}, Path=(TextElement.Foreground)}"/>
+        <Setter Property="VerticalContentAlignment" Value="Top"/>
+        <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <Setter Property="Padding" Value="0" />
+        <Setter Property="FontFamily" Value="{wpf:MaterialDesignFont}" />
```

Comment:
> This is really great work! Thank you for doing this!
> 
> A few comments:
> 
> * It would be great to pull the generic TabControl styles out for now, and submit a follow up PR. TabControl styles are a bit of a hot button topic, and I need to get more approval before that portion merges.
> 
> Some other **optional** items that could also be done later:
> 
> * Making the Divider optional. It looks like we are using a drop shadow for it now, but the spec appears to just be a simple line.
> * Include in demo app, a sample using text labels. Ideally this show default with text wrapping. It might be nice to also have the demo show the text label only appearing on the selected tab (Selected text labels example).
> * Demo app sample showing badge control use in the TabItem Header.
> * Demo app showing it used for bottom navigation

Thanks for the review!

Related to the Divider tip, I've added it. About the drop shadow, I made it inheritable by materialDesign:ShadowAssist.ShadowDepth, with a Depth2 as (old) default. Now is set to Depth0:

![image](https://user-images.githubusercontent.com/15158923/92311070-0d8b9c00-ef8a-11ea-9920-81807412ef4f.png)